app/src/main/java/org/squidfish/tuplenback/views/GameScreen.kt

```diff
@@ -17,31 +17,29 @@ import androidx.compose.material3.LinearProgressIndicator
 import androidx.compose.material3.Text
 import androidx.compose.runtime.Composable
 import androidx.compose.runtime.LaunchedEffect
-import androidx.compose.runtime.collectAsState
-import androidx.compose.runtime.getValue
 import androidx.compose.ui.Alignment
 import androidx.compose.ui.Modifier
 import androidx.compose.ui.graphics.Color
 import androidx.compose.ui.graphics.RectangleShape
 import androidx.compose.ui.tooling.preview.Preview
 import androidx.compose.ui.unit.dp
 import androidx.compose.ui.unit.sp
+import org.squidfish.tuplenback.games.Game
 import org.squidfish.tuplenback.games.GameModule
 import org.squidfish.tuplenback.games.modules.GridGame
 import org.squidfish.tuplenback.games.modules.SoundGame
-import org.squidfish.tuplenback.models.AppEvent
-import org.squidfish.tuplenback.models.GameViewModel
+import org.squidfish.tuplenback.models.GameState
 import org.squidfish.tuplenback.models.RecallCheck
 
 @Composable
 fun GameScreen(
     modifier: Modifier,
```

Comment:
`modifier` should be the first optional parameter of a composable (no need to change it now, just keep it in mind)